.github/workflows/reusable-acceptance.yml

```diff
@@ -0,0 +1,128 @@
+name: reusable-acceptance
+
+on:
+  workflow_call:
+    inputs:
+      name:
+        required: true
+        type: string
+      additional-flags:
+        required: false
+        type: string
+        default: ""
+      consul-k8s-image:
+        required: false 
+        type: string
+        default: docker.mirror.hashicorp.services/hashicorpdev/consul-k8s-control-plane:latest
+      directory:
+        required: true
+        type: string
+      go-version:
+        required: true
+        type: string
+      gotestsum-version:
+        required: true
+        type: string
+      kind-version:
+        required: false
+        type: string
+        default: "v1.22.4"
+    secrets:
```

Comment:
You have to enable the secrets on both sides of the action, as the caller and the callee.